salesforce/Ballerina.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-distribution = "2201.3.0"
+distribution = "2201.7.0"
```

Comment:
I think we need to update the [`Package.md`](https://github.com/ballerina-platform/module-ballerinax-sfdc/blob/master/salesforce/Package.md) as well